feat(native): add user feedback feature description (#9512)
getsentry/sentry-docs: docs/platforms/native/user-feedback/index.mdx

---
title: Set Up User Feedback

> description: "Learn more about collecting user feedback when an event occurs. Sentry pairs the feedback with the original event, giving you additional insight into issues."

Reviewer: ```suggestion
description: "Learn how to view user feedback submissions which, paired with the original event, give you additional insight into issues."
```

sidebar_order: 6000
---

> When a user experiences an error, Sentry provides the ability to collect additional feedback. You can collect feedback according to the method supported by the SDK.

Reviewer: ```suggestion
Sentry makes it possible to collect additional feedback when a user experiences an error. You can collect feedback according to the method supported by your SDK of choice.
```

## User Feedback API

> The user feedback API allows you to collect user feedback while utilizing your own UI. You can use the same programming language you have in your app to send user feedback. In this case, the SDK creates the HTTP request so you don't have to deal with posting data via HTTP.
> 
> Sentry pairs the feedback with the original event, giving you additional insight into issues. Sentry needs the `eventId` to be able to associate the user feedback to the corresponding event. For example, to get the `eventId`, you can use <PlatformLink to="/configuration/options/#before-send"><PlatformIdentifier name="before-send" /></PlatformLink> or the return value of the method capturing an event.
> 
> <PlatformContent includePath="user-feedback/sdk-api-example/" />
> 
> Alternatively, you can use the [User Feedback API endpoint](/api/projects/submit-user-feedback/) directly.

Reviewer: ```suggestion
The user feedback API allows you to collect user feedback using your own UI. You can use the same programming language you have in your app to then send the feedback. In this case, the SDK creates the HTTP request so you don't have to deal with posting data via HTTP.

Sentry pairs the feedback with the original event, giving you additional insight into issues. In order to do this, Sentry needs the `eventId`. For example, to get the `eventId`, you can use <PlatformLink to="/configuration/options/#before-send"><PlatformIdentifier name="before-send" /></PlatformLink> or the return value of the method capturing an event.

<PlatformContent includePath="user-feedback/sdk-api-example/" />

Alternatively, you can use the [User Feedback API endpoint](/api/projects/submit-user-feedback/) directly.
```
